### PR TITLE
Sync framework adapters and drop the runtime column

### LINKED ISSUE #676: Sync supported framework adapters
They're outdated. This needs to be fixed.

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Closes #676.

### CODE REVIEW

File: site/docs/guide/deployment-types.md [hunk 1]
```diff
@@ -241,21 +241,25 @@ Each adapter is responsible for relaying input and output from the web framework
...
+| `fastify`        | Fastify                                                                        |
+| `hono`           | Hono                                                                           |
+| `http`, `https`  | Node.js `http`/`https` modules, Vercel                                         |
+| `koa`            | Koa                                                                            |
+| `next-js`        | Next.js                                                                        |
+| `oak`            | Oak                                                                            |
+| `serveHttp`      | `Deno.serveHttp`                                                               |
+| `std/http`       | `Deno.serve`, `std/http`, `Deno.upgradeHttp`, `Fresh`, `Ultra`, `Rutt`, `Sift` |
+| `sveltekit`      | SvelteKit                                                                      |
+| `worktop`        | The `worktop` framework for Cloudflare workers                                 |
```
Comment (roj1512) [inline site/docs/guide/deployment-types.md:262]:
```suggestion
| `worktop`        | The `worktop` framework for Cloudflare Workers                                 |
```

File: site/docs/guide/deployment-types.md [hunk 2]
```diff
@@ -241,21 +241,25 @@ Each adapter is responsible for relaying input and output from the web framework
...
+| `fastify`        | Fastify                                                                        |
+| `hono`           | Hono                                                                           |
+| `http`, `https`  | Node.js `http`/`https` modules, Vercel                                         |
+| `koa`            | Koa                                                                            |
+| `next-js`        | Next.js                                                                        |
+| `oak`            | Oak                                                                            |
+| `serveHttp`      | `Deno.serveHttp`                                                               |
+| `std/http`       | `Deno.serve`, `std/http`, `Deno.upgradeHttp`, `Fresh`, `Ultra`, `Rutt`, `Sift` |
+| `sveltekit`      | SvelteKit                                                                      |
+| `worktop`        | The `worktop` framework for Cloudflare Workers                                 |
```
Comment (KnorpelSenf) [inline site/docs/guide/deployment-types.md:262]:
```suggestion
| `worktop`        | Worktop                                                      |
```

**Review by roj1512 — APPROVED** [review]
LGTM.

**Comment by KnorpelSenf** [comment]
What about https://github.com/grammyjs/website/pull/739#discussion_r1145978170?

**Review by fadzikri — APPROVED** [review]

**Review by habemuscode — APPROVED** [review]

**Review by roj1512 — APPROVED** [review]
LGTM bois.

**Pull request merged**